src/xdocs/config_imports.xml

```diff
@@ -1500,7 +1500,7 @@ import java.util.stream.IntStream;
                 Thus, the empty list of type groups (the default value) means
                 one group for all type imports.
               </td>
-              <td><a href="property_types.html#regexp">Regular Expressions</a></td>
+              <td><a href="property_types.html#Pattern">Patterns</a></td>
```

Comment:
If we have a unique type, `Pattern` versus `Patterns`, then we should have a unique section for it.

@romani ping

Reply:
name should be exact name of java class. It is ok to skip package name in table to be brief, but full name of type should be present in `property_types.html#Pattern`.

Reply:
So, I should make it `Pattern[]` right?

Reply:
it is `String[]`. This Check has special provate method on how it parse that values. Check should explain all details in description. Type should be same as in setter `String...` is fancy name of `String[]`. We should use `String[]`.

Reply:
this is moved to out to address separately. related discussion https://github.com/checkstyle/checkstyle/issues/8328#issuecomment-651952351